config/routes.rb

```diff
@@ -0,0 +1,97 @@
+Rails.application.routes.draw do
+root to: 'media#index'
+  resources :albums
+  resources :movies
+  resources :books
+  resources :media, only: [:index, :show]
```

Comment:
Since we're only using the `index` action on the media controller we should remove `:show` from this list.